docs/Report.md

```diff
@@ -31,7 +31,12 @@ header-includes: |
 
 ## Domain Model
 Provide an illustration of your domain model. Make sure that it is correct and complete. In case you are using ASP.NET Identity, make sure to illustrate that accordingly.
-<!-- ![Illustration of the _Chirp!_ data model as UML class diagram.](Images/domain_model.png) -->
+
+We haven't create a whole domain model, since our application is quite detailed, but in the next section you'll se package diagrams and a detailed onion architecture diagram, which shows how our application communicates between classes. The application that we've build is what we call a Twitter-clone. 
```

Comment:
We haven't created a whole domain model, since our application is quite detailed, but in the next section, you'll see package diagrams [@fig:CoreUML; @fig:InfrastructureUML; @fig:RazorUML; @fig:PagesUML] and a detailed onion architecture diagram [@fig:OnionClassDiagram], which shows how our application communicates between classes. The application is heavily inspired by social media platforms such as X(formerly Twitter)